### ISSUE TITLE
Sass always compiles

### ISSUE THREAD

**Issue by josemarluedke** [salsify/ember-css-modules#120]
I'm seen big rebuild times due to `SassCompiler` when changing any file (js or hbs) on an app using css-modules.

I noticed this since we started using css-modules as we migrated our app to use css-modules recently. The app is not big, we have about 2110 lines of scss code. 

Here is what the build time looks like for changing an hbs file: 

```
file changed pods/index/template.hbs

Build successful (2926ms) – Serving on http://localhost:4200/



Slowest Nodes (totalTime => 5% )              | Total (avg)
----------------------------------------------+---------------------
SassCompiler (2)                                         | 1249ms (624 ms)
BroccoliMergeTrees (30)                             | 212ms (7 ms)
Pod Templates (1)                                        | 166ms
Package /assets/vendor.js (1)                      | 158ms
```

So, the question is: How to solve this issue? Is sass always building the entirely app?

Here is my configuration: 

```
    cssModules: {
      headerModules: [
        'some-of-my-app-styles-here.scss'
      ]
    },
```

## Versions

```
➜ ember -v
ember-cli: 3.4.3
node: 10.11.0
os: darwin x64
```

Ember v3.4.4. 
ember-css-modules v1.0.1
ember-css-modules-sass v1.0.1

**Comment by dfreeman** [comment]
I'd say this qualifies as a bug: our build output should be stable if the inputs haven't changed, and this timing indicates that that's probably not the case.

**Comment by josemarluedke** [comment]
@dfreeman Any change you can give me some guidance to track where this bug might be? I'm hoping to get this problem fixed soon as it is impacting build times for every file change.

Thanks.

**Comment by dfreeman** [comment]
@josemarluedke I honestly don't have super great guidance to offer; it's been quite a while since I've debugged this kind of thing. I'd probably start in [broccoli-css-modules](https://github.com/salsify/broccoli-css-modules/) and make sure that that has stable output in the face of changes it doesn't care about (i.e. changing a `.js` in the input doesn't cause the files in the output to be written again). If that's already true, then there's likely something in the Broccoli nodes we're chaining together here in the addon that's unstable.

I'll aim to set aside a couple of hours this week to try and tackle this.

**Comment by dfreeman** [comment]
@josemarluedke This should be fixed in the just-published v1.0.3! Please feel free to reopen or ping me on Discord if you're still having any issues.

**Comment by josemarluedke** [comment]
@dfreeman Thank you so much for working on this. I really appreciate it. I can confirm that this solves the issues I was having. Thanks.
